### PR TITLE
added image component

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (13)
Screenshot (14)
Screenshot (15)

image component done, not responsive

### CODE REVIEW

File: styles/image/image.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
```
Comment (yashpatel1797) [inline styles/image/image.css:11]: not mandatory but the suggestion is you can declare this in common CSS and import that in file where you want to use.

File: styles/image/image.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.image-container{
+    padding: 1rem;
+}
+img {
+  width: 300px;
```
Comment (yashpatel1797) [inline styles/image/image.css]: here you have used rem, px try to be consistent.

**Review by yashpatel1797 — APPROVED** [review]
code looks good.

File: styles/image/image.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/image/image.css:11]: ok i will change, i have done these in all previous code. So at the time of refactoring, will do that

File: styles/image/image.css (hunk 2 above)
Comment (Sharmaryan) [reply styles/image/image.css]: done

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge!

**Review by Divya70 — APPROVED** [review]

**Comment by preethasri** [comment]
Looks good,👍

**Pull request merged**